airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/csv_parser.py

```diff
@@ -178,5 +221,17 @@ def cast_types(row: Dict[str, str], property_types: Dict[str, Any], logger: logg
     return result
 
 
+def _value_to_bool(value: str, true_values: List[str], false_values: List[str]) -> bool:
```

Comment:
Should this check be case-insensitive? Thinking about this, I feel it provides more flexibility if not but the UX might be annoying if I want to validate "true" as case insensitive (i.e. I'll need to provide "True", "TRUE", tRue", "tRUe", etc...)

The same question probably applies to null values

Reply:
I think they should be case-sensitve for backwards compatibility. We can add the option for case-insensitivity if needed.

Documented that the strings are case-sensitive in the spec